packages/path_provider/path_provider/test/path_provider_test.dart

```diff
@@ -25,6 +25,9 @@ void main() {
 
     setUp(() async {
       PathProviderPlatform.instance = MockPathProviderPlatform();
+      // This is required because we manually register the Linux path provider when on the Linux platform.
+      // Will be removed when automatic registration of dart plugins is implemented.
```

Comment:
Please link to https://github.com/flutter/flutter/issues/52267 in any comment about workarounds so that it's easy for someone without full context to trace back to the relevant discussion and current state.